run_service_ssh.ps1

```diff
@@ -0,0 +1,35 @@
+# This script first runs the ggp ssh init --instance <id> command to get
+# the needed credentials to connect via ssh. Then connects to the instance 
+# with portforwarding and starts OrbitService.
+
+
+Invoke-Expression "ggp ssh init" | Tee-Object -Variable ggp_out
```

Comment:
Is GGP always in the PATH or do we have to check the GGP_SDK_PATH environment variable?

Reply:
done